### ISSUE TITLE
[Bugfix] Fix toggle basic functionality

### ISSUE THREAD

**Issue by LJKaski** [vrk-kpa/suomifi-ui-components#154]
Can only be toggled on by dragging, cannot be toggled off. State change triggers twice when clicking the toggle in the styleguide.

**Comment by J-Kallunki** [comment]
Should base on this format:
```html
<label for="suomifi-checkbox-id" class=“suomifi-checkbox">
  <input type="checkbox" id="suomifi-checkbox-id" class=“suomifi-checkbox_input" />
  <svg class=“suomifi-checkbox_icon" />
  <span>Label for the checkbox</span>
</label>
```
```css
.suomifi-checkbox .suomifi-checkbox_input:checked + .suomifi-checkbox_icon {
}
.suomifi-checkbox .suomifi-checkbox_input:focus + .suomifi-checkbox_icon {
}
.suomifi-checkbox .suomifi-checkbox_input {
  position: absolute;
  // SVG-ikonin koko
  width: 1em;
  height: 1em;
  opacity: 0;
}
```
and use idGenerator()
